src/docfx/build/toc/parser/TableOfContentsParser.cs

```diff
@@ -137,7 +137,7 @@ private static (List<Error>, TableOfContentsModel) LoadTocModel(JToken tocToken)
                 // set resolved href back
                 tocModelItem.Href = resolvedTocHref ?? resolvedTopicHref ?? resolvedTopicItemFromTocHref?.Href;
                 tocModelItem.TocHref = resolvedTocHref;
-                tocModelItem.Homepage = resolvedTopicHref;
+                tocModelItem.Homepage = !string.IsNullOrEmpty(tocModelItem.TopicHref) ? resolvedTopicHref : null;
```

Comment:
```suggestion
                tocModelItem.Homepage = !string.IsNullOrEmpty(resolvedTopicHref) ? resolvedTopicHref : null;
```

Reply:
not that case, when resolvedTopicHref is not null, it may be resolved from `topicHref`, but also from `href`